src/ci/docker/disabled/riscv64gc-linux/Dockerfile

```diff
@@ -0,0 +1,102 @@
+# based on armhf-gnu/Dockerfile
+FROM ubuntu:20.04
+
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
+    bc \
+    bison \
+    ca-certificates \
+    cmake \
+    cpio \
+    curl \
+    debian-ports-archive-keyring \
+    debootstrap \
+    flex \
+    gcc \
+    gcc-riscv64-linux-gnu \
+    git \
+    g++-riscv64-linux-gnu \
+    g++ \
+    libc6-dev \
+    libc6-dev-riscv64-cross \
+    make \
+    patch \
+    python3 \
+    qemu-system-misc \
+    xz-utils
+
+ENV ARCH=riscv
+ENV CROSS_COMPILE=riscv64-linux-gnu-
+
+WORKDIR /build
+
+# From https://github.com/michaeljclark/busybear-linux/blob/master/conf/linux.config
+COPY riscv64gc-linux/linux.config /build
+
+# Compile the kernel that we're going to be emulating with. This is
+# basically just done to be compatible with the QEMU target that we're going
+# to be using when running tests.
+RUN curl https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-5.6.16.tar.xz | tar xJf - && \
+    cp linux.config linux-5.6.16/.config && \
+    cd /build/linux-5.6.16 && \
+    make olddefconfig && \
+    make -j$(nproc) vmlinux
+RUN cp linux-5.6.16/vmlinux /tmp
+RUN rm -rf linux-5.6.16
```

Comment:
If you use `... && make -j$(nproc) vmlinux && cp linux-5.6.16/vmlinux /tmp && rm -rf linux-5.6.16`, the `linux-5.6.16` dir will not become part of the dockee image. Every RUN command creates a new overlay on top of the image as of before the RUN command. It doesn't replace it.

Reply:
Gentle ping @tblah

Reply:
@bjorn3 thanks for the reminder, I'll take a look